src/presetsController.ts

```diff
@@ -802,18 +826,45 @@ export class PresetsController {
         }
     }
 
+    private checkCompatibility(configurePreset: preset.ConfigurePreset | null, buildPreset?: preset.BuildPreset | null, testPreset?: preset.TestPreset | null): {buildPresetCompatible: boolean; testPresetCompatible: boolean} {
```

Comment:
can we rename this to "checkPresetCompatibility"?

Reply:
The 3 inputs are presets, so I don't think we need to include "preset" in the name of the function.  I think it should be very clear what the subject of this function is.